### PR TITLE
Okta ansible

### PR DESCRIPTION
**Pull request by sudobinbash**
This blog post has the following features:

- [X] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadeveloper account (git under ASA's team account: https://github.com/okta-server-asa/asa-ansible-example)
- [X] A title that's approved by Dev Advocacy
- [X] A URL that's approved by Dev Advocacy
- [X] The content has been ran through Grammarly (https://www.grammarly.com/)
- [X] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2021-01-27-okta-ansible.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+description: "How to integrate Okta and Ansible to manage accounts in servers while abstracting account management from your playbooks"
+tags: [devops]
+tweets:
+- "Learn how to integrate Okta and Ansible to manage server accounts and abstract account management from your playbooks"
+- "Remove account management from your Ansible playbooks. Here's how"
+image: blog/
+type: conversion
+---
+
+**Tip:** This tutorial is part of our series on how to integrate Okta with popular Infrastructure as a Code solution. If you're not into Ansible, check out our [Puppet](/blog/2021/01/22/okta-puppet) and [Terraform](/blog/2020/04/24/okta-terraform-automate-identity-and-infrastructure) tutorials.
```
Comment (jpf) [inline _source/_posts/2021-01-27-okta-ansible.md:16]: Can we take this line out for now? Maybe comment it out so we can put it back in after the other posts are up?

File: _source/_posts/2021-01-27-okta-ansible.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+To overcome this, admins would typically look at distributing SSH keys or using some sort of 3rd party LDAP/AD for login.
+
+Distributing SSH Keys has major maintenance implications since it requires a lot of work anytime you need to rotate keys. If you use SSH keys for user logins, the recommendation is to rotate keys often – to reduce risks in case the keys are compromised – plus every time an admin leaves the company.
+
+Integrating your infra to an AD/LDAP server – usually using a PAM module – maybe even more challenging, since this type of server is usually monolith and hosted on-prem. Getting a robust integration here is a tall order. From site-to-site VPNs (making your AD available everywhere) to adding additional servers (keeping compatibility across a variety of server distros), this is the kind of pain no Ops Engineer would like to sign up for.
```
Comment (jpf) [inline _source/_posts/2021-01-27-okta-ansible.md:24]: s/maybe even more challenging/may be even more challenging/

Comment (jpf) [reply _source/_posts/2021-01-27-okta-ansible.md:24]: s/usually monolith/usually monolithic/

File: _source/_posts/2021-01-27-okta-ansible.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+To ssh into your server, enter `sft ssh <name-of-your-server>`:
+
+```sh
+# sft ssh asa-ansible4-ubuntu
+/home/frederico_hakamine #  
+```
+
+## Wait, I have questions
+
+**What Okta ASA does for the login?**
```
Comment (jpf) [inline _source/_posts/2021-01-27-okta-ansible.md:141]: Suggest changing to:

```
**What is Okta ASA doing?**
```

File: _source/_posts/2021-01-27-okta-ansible.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+- Okta ASA secures access to Linux and Windows servers in SSH and RDP connections using the server agent (the same one that enrolled the server in your project earlier).
+- The ASA server agent, in addition to subscribing your server to a project, also works alongside native OS features such as sudo, users, and openssh to control access during runtime and to capture any login events for audit inspection.
+- Because the agent is light and does not require firewalls and monolith LDAP or privileged access servers, it can be easily distributed across any infrastructure (IaaS, VM, or physical) and embedded in your DevOps tools.
```
Comment (jpf) [inline _source/_posts/2021-01-27-okta-ansible.md:145]: s/monolith/monolithic/

File: _source/_posts/2021-01-27-okta-ansible.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+## What's next?
+
+After testing the playbook on some hosts, you expand it to more servers. To do so, tweak the playbook. You can, for example, store the enrollment token in [Ansible's vault](https://docs.ansible.com/ansible/2.8/user_guide/vault.html), and also secure access to the Ansible controller with ASA using the same playbook. – i.e. update your `asa-playbook.yml`, changing the hosts to `local`, and then run the playbook pointing to local: `ansible-playbook  -c local -i localhost asa-playbook.yml --extra-vars "asa_enrollment_token=<local>"`.
```
Comment (jpf) [inline _source/_posts/2021-01-27-okta-ansible.md:155]: Suggest changing the last line to a triple backtick embed. So that it renders better in HTML

**Review by jpf — CHANGES_REQUESTED** [review]

File: _source/_posts/2021-01-27-okta-ansible.md (hunk 1 above)
Comment (sudobinbash) [reply _source/_posts/2021-01-27-okta-ansible.md:16]: By the time Ansible goes out, Puppet and Terraform will be already out

File: _source/_posts/2021-01-27-okta-ansible.md (hunk 2 above)
Comment (sudobinbash) [reply _source/_posts/2021-01-27-okta-ansible.md:24]: fixed

File: _source/_posts/2021-01-27-okta-ansible.md (hunk 3 above)
Comment (sudobinbash) [reply _source/_posts/2021-01-27-okta-ansible.md:141]: fixed.

**Review by jpf — APPROVED** [review]

**Pull request merged**